accepted/future-releases/abstract-external-fields/implementation-plan.md

```diff
@@ -0,0 +1,157 @@
+# Implementation Plan for "External and Abstract Variables"
```

Comment:
I don't see an entry for dartdoc in here; is there expected to be no change?

Reply:
No, it's likely that there will be some work. For instance, `///` can be placed on a declaration using new syntax. Added a section.